app/helpers/importmap/importmap_tags_helper.rb

```diff
@@ -1,6 +1,14 @@
 module Importmap::ImportmapTagsHelper
   # Setup all script tags needed to use an importmap-powered entrypoint (which defaults to application.js)
-  def javascript_importmap_tags(entry_point = "application", importmap: Rails.application.importmap)
+  def javascript_importmap_tags(entry_point = "application")
+    entry_point = entry_point.to_s
+
+    importmap_identifier =
```

Comment:
I don't importmap_identifier being used. Am I missing something?

Reply:
Did you mean to assign this to `entrypoint`?

Reply:
Yes, good catch, this should be `entry_point` instead of `importmap_identifier`.
Now that I look at it again though, it probably makes more sense to simply check for `Rails.application.importmaps.key?(entry_point)` and raise an error if this returns `false`.
The "application" importmap should always be there, and if a different value is supplied for `entry_point`, an importmap with this name should also exist.

Reply:
Fixed in new commit.